content/docs/contextual-editing/tinafield.mdx

```diff
@@ -7,7 +7,7 @@ title: The tinafield helper
 
 Tina's "click to edit" feature allows editors to select the element they want to edit on the page in order to see it in the sidebar.
 
-<CloudinaryVideo src="https://res.cloudinary.com/forestry-demo/video/upload/q_80,h_584/v1684344056/tina-io/docs/quick-edit-demo" />
+<WebmEmbed embedSrc="/video/quick-edit-demo.mp4"/>
```

Comment:
**Issue**: WebmEmbed not being rendered correctly.

![image](https://github.com/user-attachments/assets/2d6e848f-714f-4755-b50b-4bdfa739c9a3)
**Figure: Updated site on the left**

Reply:
Resolved